STYLE_GUIDE.md

```diff
@@ -170,6 +185,8 @@ All imports in composer should be absolute -- that is, they do not begin with a
     1.  It must also be specified in the `run_constrained` and the `test.requires` section.
     1.  The import must be conditionally imported in the code. For example:
 
+        <!--pytest-codeblocks:importorskip(monai)-->
+        <!--pytest-codeblocks:importorskip(scikit-learn)-->
         ```python
         from composer.utils import MissingConditionalImportError
```

Comment:
Note: This test should pass if the import exists (eg installed with `all`) and raise an error if not installed with UNet support (eg with `dev`). Unfortunately, we can't conditionally change expectation based on packages for doctests, so we just skip the failure case. Same is true for the deepspeed test